lib/index.ts

```diff
@@ -11,6 +11,7 @@ export * from './io/QuadSourceFile';
 export * from './quadmatcher/IQuadMatcher';
```

Comment:
Missing export for `TermTemplateQuadComponentCast`?

Reply:
I have split that component off into #47